week-5/A-dom-manipulation/exercise.js

```diff
@@ -23,11 +29,34 @@ Task 3
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
 
+var addParagText = document.querySelector("#addTextBtn");
+var selectButtonsDiv = document.querySelector(".buttons");
+addParagText.addEventListener("click", function() {
+  addSomeText();
+});
 
+function addSomeText() {
+  console.log("heyy");
+  var newParag = document.createElement("p");
+  selectButtonsDiv.appendChild(newParag);
+  newParag.innerText = "Read More!";
+}
 
 /*
 Task 4
 ======
 
 When the 'Larger links!' button is clicked, the text of all links on the page should increase.
-*/
\ No newline at end of file
+*/
+var largerButtons = document.querySelector("#largerLinksBtn");
+
+var selectBtn = document.getElementsByClassName("btn");
```

Comment:
You're selecting anything with the class `btn` here. However the question is asking you to change the all links. A "link" is just another word for the `<a>` element. How could you change this selector to get all `<a>` elements instead?

Reply:
ByTagName. I'll change it, too.